.ci/setup_env_fedora.sh

```diff
@@ -27,23 +28,20 @@ if grep -q "N" /sys/module/kvm_intel/parameters/nested; then
 fi
 
 sudo -E dnf -y install dnf-plugins-core
-sudo -E dnf config-manager --add-repo \
-http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/Fedora_$VERSION_ID/home:clearcontainers:clear-containers-3.repo
-
 sudo -E dnf makecache
 
 echo "Install clear containers dependencies"
 sudo -E dnf -y groupinstall "Development tools"
 sudo -E dnf -y install libtool automake autoconf bc
 
 echo "Install qemu-lite binary"
-sudo -E dnf -y install qemu-lite
+"${cidir}/install_qemu_lite.sh" "${qemu_clear_release}" "${qemu_lite_sha}" "fedora"
```

Comment:
what about changing `"fedora"` with `"$ID"` as we already sourced `/etc/os-release`